From 423138ae07d7739040d37d67eddca7d9a7ae0fb8 Mon Sep 17 00:00:00 2001
From: Alexey Makhalov <amakhalov@vmware.com>
Date: Wed, 5 Jun 2019 17:17:28 -0700
Subject: [PATCH 05/12] 9p: trans_fd: extend port variable to u32

This patch is an addition to vsock transport for 9p patch.
vsock layer uses 32 bits for the port number but trans_fd
options parser limits it to 16 bits. It is safe to extend
it since it is internal structure and does not break
existing code.
---
 net/9p/trans_fd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/9p/trans_fd.c b/net/9p/trans_fd.c
index e7ec72e..5d42d84 100644
--- a/net/9p/trans_fd.c
+++ b/net/9p/trans_fd.c
@@ -53,7 +53,7 @@ static struct p9_trans_module p9_fd_trans;
 struct p9_fd_opts {
 	int rfd;
 	int wfd;
-	u16 port;
+	u32 port;
 	bool privport;
 };
 
-- 
2.7.4

